core/optaplanner-core-impl/src/main/java/org/optaplanner/core/api/domain/variable/VariableListener.java

```diff
@@ -12,6 +12,17 @@
  */
 public interface VariableListener<Solution_, Entity_> extends AbstractVariableListener<Solution_, Entity_> {
 
+    /**
+     * When set to {@code true}, this has a slight performance loss in Planner.
+     * When set to {@code false}, it's often easier to make the listener implementation correct and fast.
```

Comment:
Let's clean this up while we're moving this javadoc.

- No point for "in Planner". Everything in the optaplanner docs in for optaplanner
- Avoid "slight", "often", etc in docs because too vague (Docs guidelines)


```suggestion
     * When set to {@code true}, this has a performance loss.
     * When set to {@code false}, it's easier to make the listener implementation correct and fast.
```